### PR TITLE
Syntax

### LINKED ISSUE #85: PBASIC Syntax Highlighting
Add PBASIC Syntax Highlighting as described in [Issue 71](https://github.com/parallaxinc/ChromeIDE/issues/71).

NOTE: True PBASIC Syntax highlighting also changes the case of the symbols in the source code, and also causes those changes to be saved in the file (as if entered that way by the user).  Since that is probably beyond what is readily available in the Chrome IDE edit control, we will gladly defer the case-modification feature for now and just request highlight and style adjustment.

### PR DESCRIPTION
**Pull request by spatializes**
#### What's this PR do?

Apply syntax colors as described in issue #71 
Targets PBASIC full syntax 
Closes #85 and #71 
#### Where should the reviewer start?

Load this branch and run ChromeIDE.
#### How should this be manually tested?

 Create a test file and enter in some test code. Try to use each of the token categories listed in issue 71 to ensure they are being properly colored.
#### Any background context you want to provide?

The highlighter is based on the VB code file. The css theme is modified from neo. 
Only piece not working as it should is the etDirective type.
#### What are the relevant tickets?

Issues #71 and #85 
#### Screenshots (if appropriate)

image
#### Questions:
- Is there a blog post?
  No
- Does the knowledge base need an update?
  No

### CODE REVIEW

File: assets/theme/neo.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/* PBASIC Full Syntax theme for codemirror */
```
Comment (monteslu) [inline assets/theme/neo.css]: rename this file to something else.  pbasic.css maybe?

Comment (spatializes) [inline assets/theme/neo.css]: just changed it and the references to parallax.css, parallax theme, etc.

**Comment by spatializes** [comment]
updated.

**Comment by PropGit** [comment]
Keep in mind that this is the PBASIC syntax, not Parallax syntax... the Spin and PASM languages for Propeller have very different syntax highlighting rules and each syntax highlighting set will need to be supported at that later time within the same app.

**Comment by urbantumbleweed** [comment]
This is a minor change and we will update the name.

Update: There is nothing to update as the theme related files are correctly named 'parallax' and syntax specific code is 'pbasic'.

**Comment by urbantumbleweed** [comment]
I jumped the gun on the my last comment and updated.  Please take a look.

**Pull request merged**